govcd/nsxt_distributed_firewall_test.go

```diff
@@ -265,7 +265,7 @@ func getRandomListOfNetworkContextProfiles(check *C, vcd *TestVCD, vdcClient *VC
 	check.Assert(err, IsNil)
 	openApiRefs := make([]types.OpenApiReference, 1)
 	for _, networkContextProfile := range networkContextProfiles {
-		if strings.Contains(networkContextProfile.Description, "ALG") {
+		if strings.Contains(networkContextProfile.Description, "ALG") || strings.Contains(networkContextProfile.Description, "includes the URL categories") {
```

Comment:
FYI @lvirbalas 
We talked about this patch.

@adambarreiro ,
Could you add a brief comment here "// skipping network context profile which has hardcoded destinations and throws error when used in firewall rules with specified destinations"

Reply:
Done!